subsys/bluetooth/services/fast_pair/fp_gatt_service.c

```diff
@@ -200,6 +200,11 @@ static int fp_gatt_rsp_notify(struct bt_conn *conn, const struct bt_gatt_attr *a
 
 static int notify_personalized_name(struct bt_conn *conn)
 {
+	if (!IS_ENABLED(CONFIG_BT_FAST_PAIR_PN)) {
```

Comment:
you can consider moving this guard after the variable declaration to make it consistent with other functions e.g. `handle_action_req`

Reply:
Good point. We agreed to keep variable declarations at the top